app/app/join/page.tsx

```
@@ -0,0 +1,500 @@
+  }
+
+  return (
+    <div className="min-h-[calc(100vh-48px)] relative">
+      <div className="absolute inset-x-0 top-0 h-48 bg-grid pointer-events-none" />
+      <div className="relative mx-auto max-w-6xl px-4 py-10">
+        {/* Header */}
+        <ScrollReveal>
+          <div className="mb-8">
+            <div className="mb-2 text-[10px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60">
+              // careers
+            </div>
```

Biome (2.3.14)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)